crates/tessellation/src/fill.rs

```diff
@@ -2206,7 +2207,7 @@ impl<'l> FillVertex<'l> {
         }
 
         if div > 1.0 {
-            for attribute in &mut self.attrib_buffer[..] {
+            for attribute in self.attrib_buffer.iter_mut() {
```

Comment:
It's nice when the IntoIter stuff figures out that [..] can be removed but when it can't I prefer `&mut foo[..]` over `foo.iter_mut()`.

Reply:
Clippy complains if I change it back to the former. It (incorectly) tells me I'm slicing redundantly. Should I disable the lint, or do we go for `iter_mut` in this particular instance instead?

Reply:
Bah, Ok let's not swim against the current. you can keep it as it is in your patch.